packages/astro/src/core/errors/errors.ts

```diff
@@ -5,6 +5,7 @@ import { getErrorDataByCode } from './utils.js';
 
 interface ErrorProperties {
 	code: AstroErrorCodes | DiagnosticCode;
+	title?: string;
```

Comment:
This new property is currently not shown in the overlay, but:
- It'll be in the new one
- It's shown in docs

So getting ready for it now